common/presenters/allocation-people-to-cards.js

```diff
@@ -0,0 +1,22 @@
+const personToCardComponent = require('./person-to-card-component')
+
+function allocationPeopleToCards({ movesCount, moves = [] }) {
+  const emptySlots = movesCount - moves.length
+  const mapper = personToCardComponent({
+    showImage: true,
+    showMeta: true,
+    showTags: true,
+  })
+  const filledSlots = moves.map(move => {
+    const { person } = move
+    if (!person.fullname) {
+      person.fullname = `${person.first_names} ${person.last_name}`
```

Comment:
Maybe have a look at how moves are formatted in the moves service. This is done as part of the response on a move to keep it in one place rather than have to do this in the presenter.

Reply:
The sole reason why this line of code exist, it's to adapt the object to the existing presenter. The `Person` service adds `fullname` to the person object, but in this case we return the data as supplied by the nested object in the api. If I do it in the service, I have to add a deep parsing of the results to add an arbitrary property that isn't supplied by the api and strictly speaking wouldn't be needed if not for this particular presenter. I am not sure it would be an improvement. If anything, we ought to remove `fullname` from usage in the presenter.

Reply:
So the reason I suggested it is that it's not just `fullname` you'll likely need.

I think there are other things like `image` that get added when formatting a person as part of a move in the move service.

I think this page will also require the use of the person's image so you'll find yourself repeating that code here. And we add support for any other properties we will also have to update it here.

Ideally it would be good to have that done as part of devour but I don't think there are available hooks to do that.